RELEASE.md

```diff
@@ -0,0 +1,128 @@
+# clever-tools releases
+
+## How is this project built, packaged and published on various repositories?
+
+* Everything is managed by a [multi-branch pipeline job in our Jenkins](https://ki2zrw1f1h-jenkins.services.clever-cloud.com/blue/organizations/jenkins/clever-tools/activity).
+* The job consists of 3 steps which are configured in the `Jenkinsfile`: **build**, **package** and **publish**.
+* We kept the `Jenkinsfile` to a minimum and put most of the build logic in several JavaScript files executed with node.js and prefixed with `job-`.
+* The `job-*.js` scripts use some external binaries when it was easier (or when there was no strong JS based solution):
+  * `tar`
+  * `zip`
+  * `git`
+  * `npm`
+  * `ssh-keyscan`
+  * `fpm` (which is a ruby gem)
+* We run this Jenkins job on an agent using this Docker image: [clever-tools-builder](https://hub.docker.com/r/clevercloud/clever-tools-builder/).
+* The `Dockerfile` to build this image is here: [docker-runner/Dockerfile](./docker-runner/Dockerfile)
+* The job is triggered on each new commit and tag via a classic Jenkins git hook setup in the GitHub project.
```

Comment:
s/tag/tagged/

Reply:
I really meant "on each new commit and each new tag".
I will clarify.